lib/src/chart/pie_chart/pie_chart_data.dart

```diff
@@ -268,6 +273,7 @@ class PieChartSectionData {
         b.badgePositionPercentageOffset,
         t,
       ),
+      shadow: b.shadow,
```

Comment:
Can we have a `FlShadowData.lerp()` here? It makes it good when animating between states.